packages/workers-shared/asset-server-worker/src/index.ts

```diff
@@ -1,5 +1,14 @@
+interface Env {
+	ASSETS_MANIFEST: ArrayBuffer;
+	ASSETS_KV_NAMESPACE: KVNamespace;
+}
+
 export default {
-	async fetch() {
-		return new Response("Hello from Asset Server Worker 🚀");
+	async fetch(request: Request, env: Env) {
+		const url = new URL(request.url);
+		const { pathname } = url;
+
+		const content = await env.ASSETS_KV_NAMESPACE.get(pathname);
```

Comment:
So in this worker we need to make use of the manifest binding.
For this PR I think it is enough to assert that the binding is of the special form that indicates it is dev mode pass through, and then to create a class that has a `get()` function that is the identity.